src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -216,17 +216,17 @@ limitations under the License.
       </description>
       <arg name="level" type="INT8U"/>
       <arg name="transitionTime" type="INT16U"/>
-      <arg name="optionMask" type="BITMAP8" introducedIn="zcl6-errata-14-0129-15" optional="1"/>
-      <arg name="optionOverride" type="BITMAP8" introducedIn="zcl6-errata-14-0129-15" optional="1"/>
+      <arg name="optionsMask" type="BITMAP8" optional="1"/>
+      <arg name="optionsOverride" type="BITMAP8" optional="1"/>
```

Comment:
```suggestion
      <arg name="OptionsMask" type="BITMAP8"/>
      <arg name="OptionsOverride" type="BITMAP8"/>
```
to match the spec....  I really wish people had actually compared this cluster XML to the spec.  :(

Reply:
In this case I was somewhat torn: the cluster does use a different capitalization than the spec, leaving the camelCase capitalization makes it consistent in the code and the names of those parameters do not impact interop.  Happy to change this to match the spec; in that case I would recommend changing the capitalization of the rest of the parameters in this cluster.  WDYT, @bzbarsky-apple ?

Reply:
We should match the spec, but do the fields this PR is not touching in a separate PR to keep scope sane.  https://github.com/project-chip/connectedhomeip/pull/21361 will do that, as well as fix a bunch of places where things are not marked nullable, etc.

Reply:
But yes, the names are not a spec issue, and _might_ not even affect our code-generated APIs in this specific capitalization aspect.

Reply:
Should I abandon this change in light of the more expansive changes in #21361 ? I fear we are just creating merge conflicts and extra work.

Reply:
You should make this change.  That change has a _lot_ of work to get it in shape and can rebase on top of this without much trouble imo.